index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://use.typekit.net/fhu4nhd.css">
+    <title>Mobile Portfolio</title>
+</head>
+<body>
+  <!--Toolbar-->
+  <header>
+    <p class="logo">My Logo</p>
+    <nav>
+      <i class="logo material-icons">
+        menu
+      </i>
+    </nav>
+  </header>
+  <section class="intro">   
+    <div class="space"></div>
+      <h1 class="prmrytxt1">
+      I’m Sir Malcom<br>Glad to see<br>you!
+    </h1>
+    <p class="prmrytxt2">
+      I’m a software developer! I can help you build a product, 
+      feature or website Look through some of my work and experience! 
+      If you like what you see and have a project you need coded, 
+      don’t hesitate to contact me.
+    </p>
+  <div class="letsconnect">
+      LET’S CONNECT
+  </div>
```

Comment:
- I believe your `LET’S CONNECT` is meant to be a clickable link and should be in either a button or an anchor tag as the `a` tag will let you connect to other sections of the website, using href. 🌸

Reply:
Thank you for your review.